stdio/scanf.c

```diff
@@ -318,7 +320,10 @@ static int scanf_parse(char *ccltab, const char *inp, int *inr, char const *fmt0
 					else {
 						*va_arg(ap, int *) = nread;
 					}
-					continue;
+					break;
```

Comment:
This works because `c == 'n'` so it won't match any of `CT_*` values in the conversion `switch` below. Whitespace will still be skipped anyway. Maybe there should be `CT_NONE` value which performs no conversion is used for `'n'` and `default` cases, together with `NOSKIP`?

Reply:
After my changes `c` is not used in the switch statement and `CT_NONE` is used for handling `%%` and `%n`. Default case handles unknown conversion specifiers by ending the conversion (behavior is undefined [according to documentation](https://pubs.opengroup.org/onlinepubs/9699919799/functions/fscanf.html), this is what glibc does).